src/Http/Offer/PriceInfoValidatingRequestBodyParser.php

```diff
@@ -33,20 +33,20 @@ public function parse(ServerRequestInterface $request): ServerRequestInterface
     /*
      * @return SchemaError[]
      */
-    private function getSchemaErrors(array $priceInfo): array
+    private function getSchemaErrors(array $priceInfos): array
     {
         $errors = [];
         $priceMatrix = [];
```

Comment:
After changing the variable names, I wonder this is not a `$nameMatrix` instead of a `$priceMatrix`?